### PR TITLE
Notebook - CIRC23 Learning the Underlying Physics of a Simulation Model of the Ocean's Temperature   

### LINKED ISSUE #10: [REVIEW] A Sensitivity Analysis of a Regression Model of Ocean Temperature
# Notebook Review: Issue alan-turing-institute/notebooks-reviews#5 

<p align="left">
    <a href="https://notebooks.gesis.org/binder/v2/gh/eds-book-gallery/repro-challenge-team-3/review?labpath=notebooks/notebook.ipynb">
        Binder
    </a>
</p>

**Submitting author:** @garimamalhotra @jedpe @dapivei

**Repository:** https://github.com/eds-book-gallery/repro-challenge-team-3

**Paper:** https://doi.org/10.1017/eds.2022.10

**Editor:** @annefou

**Reviewer:** @oscarbau @RachelFurner @ricardobarroslourenco

**Managing EiC:** @acocac

## Status

**Reviewer instructions & questions**

Hi ~@christopheredsall~ @oscarbau @RachelFurner @ricardobarroslourenco, please carry out your review in this issue by updating the checklist below. 

As a reviewer, you contribute to the technical quality of the content published by our community. 

Before the review, EiC checked if the submission fits the minimum requirements. 

The quality of the proposed contribution can be assessed through scientific, technical and code criteria. 

The reviewer guidelines are available here: https://edsbook.org/publishing/guidelines/guidelines-reviewers.html. 
Any questions/concerns please let @annefou know.

## Review checklist for @christopheredsall (replaced by @ricardobarroslourenco)

*Please check off boxes as applicable, and elaborate in comments below. Your review is not limited to these topics, as described in the reviewer guide.*

### Conflict of interest
- [x] As the reviewer I confirm that there are no conflicts of interest for me to review this work (If you are unsure whether you are in conflict, please speak to your editor _before_ starting your review).

### Code of conduct an peer-review principles
- [x] I confirm that I read and will adhere to the Environmental Data Science [code of conduct](https://github.com/alan-turing-institute/environmental-ds-book/blob/master/CODE_OF_CONDUCT.md).
- [x] I confirm that I read and will adhere to [Open Science Peer Review Oath](https://doi.org/10.12688/f1000research.5686.2) 

### General checks
- [x] **Notebook:** Is the notebook file (``notebook.ipynb``) part of the notebook repository?
- [ ] **Contribution and authorship:** Does the author list seem appropriate and complete (full name, affiliation, and GitHub/ORCID handle)?
- [x] **Scope and eligibility:** Does the submission contain an original and complete analysis according to the [scope of EDS book](https://edsbook.org/notebooks/about/aims-and-scope.html#aims-and-scope)?

### Reproducibility
- [ ] Does the notebook run in a local environment?
- [x] alan-turing-institute/environmental-ds-book#181
- [ ] Are all data sources openly accessible and properly cited (e.g. with citation to a persistent DOI) in the heading section?

### Pedagogy
- [ ] Are the notebook purpose and highlights clear?
- [ ] Does the notebook demonstrate some specific data analysis or visualisation techniques?
- [ ] Is the notebook well documented, using both markdown cells and comments in code cells?
- [ ] Does the conclusion section provide clear and concise final say on the tools, analysis and/or datasets used?
- [ ] Is the notebook narrative well written (it does not require editing for structure, language, or writing quality)?

### Ethical
- [ ] Is any linkage of datasets in the notebook unlikely to lead to an increased risk of the personal identification of individuals?
- [ ] Is the notebook truthful and clear about any limitations of the analysis (and potential biases in data and/or tools)?
- [ ] Is the notebook unlikely to lead to negative social outcomes, such as (but not limited to) increasing discrimination or injustice?

### Other Requirements
- [ ] All mentioned software should be formally and consistently cited wherever possible.
- [ ] Acronyms should be spelled out upon first mention.
- [ ] License conditions on images and figures must be respected (Creative Commons, etc.).

### Final approval (post-review)

- [ ] **Authors has responded to my review and made changes to my satisfaction. I recommend approving the notebook for publication.**~~

## Review checklist for @oscarbau

*Please check off boxes as applicable, and elaborate in comments below. Your review is not limited to these topics, as described in the reviewer guide.*

### Conflict of interest
- [x] As the reviewer I confirm that there are no conflicts of interest for me to review this work (If you are unsure whether you are in conflict, please speak to your editor _before_ starting your review).

### Code of conduct an peer-review principles
- [x] I confirm that I read and will adhere to the Environmental Data Science [code of conduct](https://github.com/alan-turing-institute/environmental-ds-book/blob/master/CODE_OF_CONDUCT.md).
- [x] I confirm that I read and will adhere to [Open Science Peer Review Oath](https://doi.org/10.12688/f1000research.5686.2) 

### General checks
- [x] **Notebook:** Is the notebook file (``notebook.ipynb``) part of the notebook repository?
- [x] **Contribution and authorship:** Does the author list seem appropriate and complete (full name, affiliation, and GitHub/ORCID handle)?
- [x] **Scope and eligibility:** Does the submission contain an original and complete analysis according to the [scope of EDS book](https://edsbook.org/notebooks/about/aims-and-scope.html#aims-and-scope)?

### Reproducibility
- [x] Does the notebook run in a local environment? See comment [#9](https://github.com/eds-book-gallery/repro-challenge-team-3/issues/9)
- [x] Does the notebook build and run in binder?
- [x] Are all data sources openly accessible and properly cited (e.g. with citation to a persistent DOI) in the heading section?

### Pedagogy
- [x] Are the notebook purpose and highlights clear?
- [x] Does the notebook demonstrate some specific data analysis or visualisation techniques?
- [x] Is the notebook well documented, using both markdown cells and comments in code cells?
- [x] Does the conclusion section provide clear and concise final say on the tools, analysis and/or datasets used?
- [x] Is the notebook narrative well written (it does not require editing for structure, language, or writing quality)?

### Ethical
- [x] Is any linkage of datasets in the notebook unlikely to lead to an increased risk of the personal identification of individuals?
- [x] Is the notebook truthful and clear about any limitations of the analysis (and potential biases in data and/or tools)?
- [x] Is the notebook unlikely to lead to negative social outcomes, such as (but not limited to) increasing discrimination or injustice?

### Other Requirements
- [x] All mentioned software should be formally and consistently cited wherever possible.
- [x] Acronyms should be spelled out upon first mention.
- [x] License conditions on images and figures must be respected (Creative Commons, etc.).

### Final approval (post-review)

- [x] **Authors has responded to my review and made changes to my satisfaction. I recommend approving the notebook for publication.**

## Review checklist for @RachelFurner

*Please check off boxes as applicable, and elaborate in comments below. Your review is not limited to these topics, as described in the reviewer guide.*

### Conflict of interest
- [x] As the reviewer I confirm that there are no conflicts of interest for me to review this work (If you are unsure whether you are in conflict, please speak to your editor _before_ starting your review).

### Code of conduct an peer-review principles
- [x] I confirm that I read and will adhere to the Environmental Data Science [code of conduct](https://github.com/alan-turing-institute/environmental-ds-book/blob/master/CODE_OF_CONDUCT.md).
- [x] I confirm that I read and will adhere to [Open Science Peer Review Oath](https://doi.org/10.12688/f1000research.5686.2) 

### General checks
- [x] **Notebook:** Is the notebook file (``notebook.ipynb``) part of the notebook repository?
- [x] **Contribution and authorship:** Does the author list seem appropriate and complete (full name, affiliation, and GitHub/ORCID handle)?
- [x] **Scope and eligibility:** Does the submission contain an original and complete analysis according to the [scope of EDS book](https://edsbook.org/notebooks/about/aims-and-scope.html#aims-and-scope)?

### Reproducibility
- [x] Does the notebook run in a local environment?
- [ ] Does the notebook build and run in binder?
- [x] Are all data sources openly accessible and properly cited (e.g. with citation to a persistent DOI) in the heading section?

### Pedagogy
- [x] Are the notebook purpose and highlights clear?
- [x] Does the notebook demonstrate some specific data analysis or visualisation techniques?
- [x] Is the notebook well documented, using both markdown cells and comments in code cells?
- [x] Does the conclusion section provide clear and concise final say on the tools, analysis and/or datasets used?
- [x] Is the notebook narrative well written (it does not require editing for structure, language, or writing quality)?

### Ethical
- [x] Is any linkage of datasets in the notebook unlikely to lead to an increased risk of the personal identification of individuals?
- [x] Is the notebook truthful and clear about any limitations of the analysis (and potential biases in data and/or tools)?
- [x] Is the notebook unlikely to lead to negative social outcomes, such as (but not limited to) increasing discrimination or injustice?

### Other Requirements
- [x] All mentioned software should be formally and consistently cited wherever possible.
- [x] Acronyms should be spelled out upon first mention.
- [x] License conditions on images and figures must be respected (Creative Commons, etc.).

### Final approval (post-review)

- [x] **Authors has responded to my review and made changes to my satisfaction. I recommend approving the notebook for publication.**

## Review checklist for @ricardobarroslourenco (replacing @christopheredsall)

*Please check off boxes as applicable, and elaborate in comments below. Your review is not limited to these topics, as described in the reviewer guide.*

### Conflict of interest
- [x] As the reviewer I confirm that there are no conflicts of interest for me to review this work (If you are unsure whether you are in conflict, please speak to your editor _before_ starting your review).

### Code of conduct an peer-review principles
- [x] I confirm that I read and will adhere to the Environmental Data Science [code of conduct](https://github.com/alan-turing-institute/environmental-ds-book/blob/master/CODE_OF_CONDUCT.md).
- [x] I confirm that I read and will adhere to [Open Science Peer Review Oath](https://doi.org/10.12688/f1000research.5686.2) 

### General checks
- [x] **Notebook:** Is the notebook file (``notebook.ipynb``) part of the notebook repository?
- [x] alan-turing-institute/environmental-ds-book#179
- [x] **Scope and eligibility:** Does the submission contain an original and complete analysis according to the [scope of EDS book](https://edsbook.org/notebooks/about/aims-and-scope.html#aims-and-scope)?

### Reproducibility
- [x] Does the notebook run in a local environment?
- [x] alan-turing-institute/environmental-ds-book#180
- [x] Are all data sources openly accessible and properly cited (e.g. with citation to a persistent DOI) in the heading section?

### Pedagogy
- [x] Are the notebook purpose and highlights clear?
- [x] Does the notebook demonstrate some specific data analysis or visualisation techniques?
- [x] Is the notebook well documented, using both markdown cells and comments in code cells?
- [x] Does the conclusion section provide clear and concise final say on the tools, analysis and/or datasets used?
- [x] Is the notebook narrative well written (it does not require editing for structure, language, or writing quality)?

### Ethical
- [x] Is any linkage of datasets in the notebook unlikely to lead to an increased risk of the personal identification of individuals?
- [x] Is the notebook truthful and clear about any limitations of the analysis (and potential biases in data and/or tools)?
- [x] Is the notebook unlikely to lead to negative social outcomes, such as (but not limited to) increasing discrimination or injustice?

### Other Requirements
- [x] All mentioned software should be formally and consistently cited wherever possible.
- [x] Acronyms should be spelled out upon first mention.
- [x] License conditions on images and figures must be respected (Creative Commons, etc.).

### Final approval (post-review)

- [x] **Authors has responded to my review and made changes to my satisfaction. I recommend approving the notebook for publication.**

## Additional instructions

Reviewer general comments are welcome on this REVIEW issue or directly to the notebook repository.

If you do the latter, you will find a Pull Request titled REVIEW where you can carry out the discussion with authors through ReviewNB, a third-party plugin in GitHub for displaying and commenting Jupyter Notebooks (see further details [here](https://edsbook.org/notebooks/about/notebooks-technologies.html#reviewnb)). 

In addition to ReviewNB, we suggest to explore or run the notebook in:
* **Binder** (run): Click the Launch Binder button at the top level of this message.

### LINKED ISSUE #172: [REVIEW] A Sensitivity Analysis of a Regression Model of Ocean Temperature
# Notebook Review: Issue #170 

<p align="left">
    <a href="https://notebooks.gesis.org/binder/v2/gh/eds-book-gallery/repro-challenge-team-3/review?labpath=notebooks/notebook.ipynb">
        Binder
    </a>
</p>

**Submitting author:** @garimamalhotra @jedpe @dapivei

**Repository:** https://github.com/eds-book-gallery/repro-challenge-team-3

**Paper:** https://doi.org/10.1017/eds.2022.10

**Editor:** @annefou

**Reviewer:** @oscarbau @RachelFurner @ricardobarroslourenco

**Managing EiC:** @acocac

## Status

**Reviewer instructions & questions**

Hi ~@christopheredsall~ @oscarbau @RachelFurner @ricardobarroslourenco, please carry out your review in this issue by updating the checklist below. 

As a reviewer, you contribute to the technical quality of the content published by our community. 

Before the review, EiC checked if the submission fits the minimum requirements. 

The quality of the proposed contribution can be assessed through scientific, technical and code criteria. 

The reviewer guidelines are available here: https://edsbook.org/publishing/guidelines/guidelines-reviewers.html. 
Any questions/concerns please let @annefou know.

## Review checklist for @christopheredsall (replaced by @ricardobarroslourenco)

*Please check off boxes as applicable, and elaborate in comments below. Your review is not limited to these topics, as described in the reviewer guide.*

### Conflict of interest
- [x] As the reviewer I confirm that there are no conflicts of interest for me to review this work (If you are unsure whether you are in conflict, please speak to your editor _before_ starting your review).

### Code of conduct an peer-review principles
- [x] I confirm that I read and will adhere to the Environmental Data Science [code of conduct](https://github.com/alan-turing-institute/environmental-ds-book/blob/master/CODE_OF_CONDUCT.md).
- [x] I confirm that I read and will adhere to [Open Science Peer Review Oath](https://doi.org/10.12688/f1000research.5686.2) 

### General checks
- [x] **Notebook:** Is the notebook file (``notebook.ipynb``) part of the notebook repository?
- [ ] **Contribution and authorship:** Does the author list seem appropriate and complete (full name, affiliation, and GitHub/ORCID handle)?
- [x] **Scope and eligibility:** Does the submission contain an original and complete analysis according to the [scope of EDS book](https://edsbook.org/notebooks/about/aims-and-scope.html#aims-and-scope)?

### Reproducibility
- [ ] Does the notebook run in a local environment?
- [x] #181
- [ ] Are all data sources openly accessible and properly cited (e.g. with citation to a persistent DOI) in the heading section?

### Pedagogy
- [ ] Are the notebook purpose and highlights clear?
- [ ] Does the notebook demonstrate some specific data analysis or visualisation techniques?
- [ ] Is the notebook well documented, using both markdown cells and comments in code cells?
- [ ] Does the conclusion section provide clear and concise final say on the tools, analysis and/or datasets used?
- [ ] Is the notebook narrative well written (it does not require editing for structure, language, or writing quality)?

### Ethical
- [ ] Is any linkage of datasets in the notebook unlikely to lead to an increased risk of the personal identification of individuals?
- [ ] Is the notebook truthful and clear about any limitations of the analysis (and potential biases in data and/or tools)?
- [ ] Is the notebook unlikely to lead to negative social outcomes, such as (but not limited to) increasing discrimination or injustice?

### Other Requirements
- [ ] All mentioned software should be formally and consistently cited wherever possible.
- [ ] Acronyms should be spelled out upon first mention.
- [ ] License conditions on images and figures must be respected (Creative Commons, etc.).

### Final approval (post-review)

- [ ] **Authors has responded to my review and made changes to my satisfaction. I recommend approving the notebook for publication.**~~

## Review checklist for @oscarbau

*Please check off boxes as applicable, and elaborate in comments below. Your review is not limited to these topics, as described in the reviewer guide.*

### Conflict of interest
- [x] As the reviewer I confirm that there are no conflicts of interest for me to review this work (If you are unsure whether you are in conflict, please speak to your editor _before_ starting your review).

### Code of conduct an peer-review principles
- [x] I confirm that I read and will adhere to the Environmental Data Science [code of conduct](https://github.com/alan-turing-institute/environmental-ds-book/blob/master/CODE_OF_CONDUCT.md).
- [x] I confirm that I read and will adhere to [Open Science Peer Review Oath](https://doi.org/10.12688/f1000research.5686.2) 

### General checks
- [x] **Notebook:** Is the notebook file (``notebook.ipynb``) part of the notebook repository?
- [x] **Contribution and authorship:** Does the author list seem appropriate and complete (full name, affiliation, and GitHub/ORCID handle)?
- [x] **Scope and eligibility:** Does the submission contain an original and complete analysis according to the [scope of EDS book](https://edsbook.org/notebooks/about/aims-and-scope.html#aims-and-scope)?

### Reproducibility
- [x] Does the notebook run in a local environment? See comment [#9](https://github.com/eds-book-gallery/repro-challenge-team-3/issues/9)
- [x] Does the notebook build and run in binder?
- [x] Are all data sources openly accessible and properly cited (e.g. with citation to a persistent DOI) in the heading section?

### Pedagogy
- [x] Are the notebook purpose and highlights clear?
- [x] Does the notebook demonstrate some specific data analysis or visualisation techniques?
- [x] Is the notebook well documented, using both markdown cells and comments in code cells?
- [x] Does the conclusion section provide clear and concise final say on the tools, analysis and/or datasets used?
- [x] Is the notebook narrative well written (it does not require editing for structure, language, or writing quality)?

### Ethical
- [x] Is any linkage of datasets in the notebook unlikely to lead to an increased risk of the personal identification of individuals?
- [x] Is the notebook truthful and clear about any limitations of the analysis (and potential biases in data and/or tools)?
- [x] Is the notebook unlikely to lead to negative social outcomes, such as (but not limited to) increasing discrimination or injustice?

### Other Requirements
- [x] All mentioned software should be formally and consistently cited wherever possible.
- [x] Acronyms should be spelled out upon first mention.
- [x] License conditions on images and figures must be respected (Creative Commons, etc.).

### Final approval (post-review)

- [x] **Authors has responded to my review and made changes to my satisfaction. I recommend approving the notebook for publication.**

## Review checklist for @RachelFurner

*Please check off boxes as applicable, and elaborate in comments below. Your review is not limited to these topics, as described in the reviewer guide.*

### Conflict of interest
- [x] As the reviewer I confirm that there are no conflicts of interest for me to review this work (If you are unsure whether you are in conflict, please speak to your editor _before_ starting your review).

### Code of conduct an peer-review principles
- [x] I confirm that I read and will adhere to the Environmental Data Science [code of conduct](https://github.com/alan-turing-institute/environmental-ds-book/blob/master/CODE_OF_CONDUCT.md).
- [x] I confirm that I read and will adhere to [Open Science Peer Review Oath](https://doi.org/10.12688/f1000research.5686.2) 

### General checks
- [x] **Notebook:** Is the notebook file (``notebook.ipynb``) part of the notebook repository?
- [x] **Contribution and authorship:** Does the author list seem appropriate and complete (full name, affiliation, and GitHub/ORCID handle)?
- [x] **Scope and eligibility:** Does the submission contain an original and complete analysis according to the [scope of EDS book](https://edsbook.org/notebooks/about/aims-and-scope.html#aims-and-scope)?

### Reproducibility
- [x] Does the notebook run in a local environment?
- [ ] Does the notebook build and run in binder?
- [x] Are all data sources openly accessible and properly cited (e.g. with citation to a persistent DOI) in the heading section?

### Pedagogy
- [x] Are the notebook purpose and highlights clear?
- [x] Does the notebook demonstrate some specific data analysis or visualisation techniques?
- [x] Is the notebook well documented, using both markdown cells and comments in code cells?
- [x] Does the conclusion section provide clear and concise final say on the tools, analysis and/or datasets used?
- [x] Is the notebook narrative well written (it does not require editing for structure, language, or writing quality)?

### Ethical
- [x] Is any linkage of datasets in the notebook unlikely to lead to an increased risk of the personal identification of individuals?
- [x] Is the notebook truthful and clear about any limitations of the analysis (and potential biases in data and/or tools)?
- [x] Is the notebook unlikely to lead to negative social outcomes, such as (but not limited to) increasing discrimination or injustice?

### Other Requirements
- [x] All mentioned software should be formally and consistently cited wherever possible.
- [x] Acronyms should be spelled out upon first mention.
- [x] License conditions on images and figures must be respected (Creative Commons, etc.).

### Final approval (post-review)

- [x] **Authors has responded to my review and made changes to my satisfaction. I recommend approving the notebook for publication.**

## Review checklist for @ricardobarroslourenco (replacing @christopheredsall)

*Please check off boxes as applicable, and elaborate in comments below. Your review is not limited to these topics, as described in the reviewer guide.*

### Conflict of interest
- [x] As the reviewer I confirm that there are no conflicts of interest for me to review this work (If you are unsure whether you are in conflict, please speak to your editor _before_ starting your review).

### Code of conduct an peer-review principles
- [x] I confirm that I read and will adhere to the Environmental Data Science [code of conduct](https://github.com/alan-turing-institute/environmental-ds-book/blob/master/CODE_OF_CONDUCT.md).
- [x] I confirm that I read and will adhere to [Open Science Peer Review Oath](https://doi.org/10.12688/f1000research.5686.2) 

### General checks
- [x] **Notebook:** Is the notebook file (``notebook.ipynb``) part of the notebook repository?
- [x] #179
- [x] **Scope and eligibility:** Does the submission contain an original and complete analysis according to the [scope of EDS book](https://edsbook.org/notebooks/about/aims-and-scope.html#aims-and-scope)?

### Reproducibility
- [x] Does the notebook run in a local environment?
- [x] #180
- [x] Are all data sources openly accessible and properly cited (e.g. with citation to a persistent DOI) in the heading section?

### Pedagogy
- [x] Are the notebook purpose and highlights clear?
- [x] Does the notebook demonstrate some specific data analysis or visualisation techniques?
- [x] Is the notebook well documented, using both markdown cells and comments in code cells?
- [x] Does the conclusion section provide clear and concise final say on the tools, analysis and/or datasets used?
- [x] Is the notebook narrative well written (it does not require editing for structure, language, or writing quality)?

### Ethical
- [x] Is any linkage of datasets in the notebook unlikely to lead to an increased risk of the personal identification of individuals?
- [x] Is the notebook truthful and clear about any limitations of the analysis (and potential biases in data and/or tools)?
- [x] Is the notebook unlikely to lead to negative social outcomes, such as (but not limited to) increasing discrimination or injustice?

### Other Requirements
- [x] All mentioned software should be formally and consistently cited wherever possible.
- [x] Acronyms should be spelled out upon first mention.
- [x] License conditions on images and figures must be respected (Creative Commons, etc.).

### Final approval (post-review)

- [x] **Authors has responded to my review and made changes to my satisfaction. I recommend approving the notebook for publication.**

## Additional instructions

Reviewer general comments are welcome on this REVIEW issue or directly to the notebook repository.

If you do the latter, you will find a Pull Request titled REVIEW where you can carry out the discussion with authors through ReviewNB, a third-party plugin in GitHub for displaying and commenting Jupyter Notebooks (see further details [here](https://edsbook.org/notebooks/about/notebooks-technologies.html#reviewnb)). 

In addition to ReviewNB, we suggest to explore or run the notebook in:
* **Binder** (run): Click the Launch Binder button at the top level of this message.

### PR DESCRIPTION
**Pull request by acocac**
### Summary

Postprint version from a notebook submission of the Climate Informatics Reproducibility Challenge (Fixes alan-turing-institute/notebooks-reviews#10)

### List of changes proposed in this PR (pull-request)

Before review:
* [x] Add submodule from the notebook repo
* [x] Add new submodule path to `_toc.yml`
* [x] Add card of the new notebook
* [x] Add references
* [x] Hide code cell content of some sections e.g. Libraries, Tables, Last tested
* [x] Right-panel looks ok (headings)

Once reviewed/approved:
* [x] Replace binder url, postprint to main
* [x] Replace submodule branch, preview to render
* [x] Generate and add badges of DOI in the notebook file and gallery
* [x] Add authors/reviewers via all-contributors
 
#### Authors
- [ ] Check the rendered version in the [Netlify Preview](https://deploy-preview-184--the-environmental-ds-book.netlify.app/notebooks/gallery/3286b92f-4fae-4cc6-a29e-e408bc844542/notebook) looks ok.
- [ ] Check the [card/thumbnail of the notebook](https://deploy-preview-184--the-environmental-ds-book.netlify.app/notebooks/gallery) is correctly displayed in the Gallery. Are the suggested short notebook title in the gallery and left menu ok?

#### Maintainers
- [ ] Check badges in the gallery and notebook file
- [ ] Check binder

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file: 

**Authors**
  - [x] @garimamalhotra
  - [x] @jedpe
  - [x] @dapivei
    
**Reviewers**
  - [x] @oscarbau
  - [x] @RachelFurner 
  - [x] @ricardobarroslourenco

### CODE REVIEW

**Comment by acocac** [comment]
Hi @garimamalhotra @jedpe @dapivei,

This PR aims to provide a preview of the post-print version of your notebook submission. Feel free to explore it by clicking on the binder badge. We also appreciate if any of you complete the reviewers checklist in the PR description.

Note the proposed version has some changes that facilitate running the notebook in a standard binder. Instead of downloading the large files from zenodo, they're fetched from a cloud storage. The adapted source code can be found in the [post-print branch](https://github.com/eds-book-gallery/3286b92f-4fae-4cc6-a29e-e408bc844542/tree/postprint). Most of the new scripts in `src` have the suffix `_cloud`. The original notebook file which downloads the input dataset and models from zenodo repositories remains in the notebook folder. 

Additional suggestions would be welcome. If everything looks ok, we aim to publish the notebook between Tuesday and Friday next week 🗓️

File: misc/rohub/register.py
Comment (dapivei) [inline misc/rohub/register.py:1]: Added my ORCID number through the following pull request (https://github.com/alan-turing-institute/environmental-ds-book/pull/187)

**Review by dapivei — COMMENTED** [review]
Reviewed: 

- [X] Rendered version in the [Netlify Preview](https://deploy-preview-184--the-environmental-ds-book.netlify.app/notebooks/gallery/3286b92f-4fae-4cc6-a29e-e408bc844542/notebook). Everything looks ok.

- [X] [Card/thumbnail of the notebook](https://deploy-preview-184--the-environmental-ds-book.netlify.app/notebooks/gallery) is correctly displayed in the Gallery. 
- [X] The suggested short notebook title in the gallery and left menu is ok.
- [X] Badges in the gallery and notebook file are ok.
- [X] Binder is **not** working properly, I waited almost 30 minutes and could not access the interactive version of the notebook.

File: misc/rohub/register.py
Comment (acocac) [reply misc/rohub/register.py:1]: @dapivei thanks for adding the ORCID number.

**Comment by acocac** [comment]
> * [x]  Binder is **not** working properly, I waited almost 30 minutes and could not access the interactive version of the notebook.

This is weird, I just opened it in a private tab and the notebook runs in Binder. I'll try to run it when merged to the main branch of the notebook repository.

**Comment by garimamalhotra** [comment]
1. Card/thumbnail looks ok.
2. I also had issues running the binder. I tried it quite a few times but It crashes when I run the model training cell.
3. Rendered version in the Netlify -  In Figure 4, for the coefficient analysis, the text at the end - 'For Figure 4a... For Figure 4b.... For Figure 4c...' is hard to read because of the long sentences and the scrolling required. I would suggest that we change its format so that is similar to the paragraph above 'High-weighted inputs ..'

**Comment by acocac** [comment]
> * I also had issues running the binder. I tried it quite a few times but It crashes when I run the model training cell.

Minor changes in `ReadMITGCM` to release memory. The model training cell should run.

> * Rendered version in the Netlify -  In Figure 4, for the coefficient analysis, the text at the end - 'For Figure 4a... For Figure 4b.... For Figure 4c...' is hard to read because of the long sentences and the scrolling required. I would suggest that we change its format so that is similar to the paragraph above 'High-weighted inputs ..'

Changed the style to lettered list using HTML. This was implemented in other sections with similar lists.

**Comment by acocac** [comment]
@all-contributors please add @garimamalhotra for blog, code

**Comment by acocac** [comment]
@all-contributors please add @jedpe for blog, code

**Comment by acocac** [comment]
@all-contributors please add @dapivei for blog, code

**Comment by acocac** [comment]
@all-contributors please add @oscarbau for review

**Comment by acocac** [comment]
@all-contributors please add @RachelFurner for review

**Comment by acocac** [comment]
@all-contributors please add @ricardobarroslourenco for review

**Comment by acocac** [comment]
@annefou can you please complete the checklist for maintainers and approve the PR if everything looks ok?   

Can you also double check the notebook works in Binder? I haven't had any issues, but it seems authors have reported some when launching Binder through the notebook badge or running the model training cell.

**Review by annefou — APPROVED** [review]
Awesome! So nice to see this notebook in the EDSBook! Thank you!

**Pull request merged**
